builtins/amp-ad.js

```diff
@@ -283,8 +283,17 @@ export function installAd(win) {
      * @private
      */
     noContentHandler_() {
-      this.element.removeChild(this.iframe_);
-      this.toggleFallback(true);
+      if (!this.fallback_) {
+        this.changeHeight(0, () => {
```

Comment:
It would be important to do it in the same instant to reduce the number of style recalcs and layouts to 1. Setting display to none is a much cheaper op than setting height to 0, because the former doesn't require layouting the children.